.travis/publish_multiarch.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -e
+REGISTRY="quay.io"
+
+PUBLISH_IMAGES_LIST=(
+  eclipse/che-theia-dev
+  eclipse/che-theia
+  eclipse/che-theia-endpoint-runtime-binary
+  eclipse/che-theia-vsix-installer
+)
+
+SHORT_SHA=$(git rev-parse --short HEAD)
+
+for image in "${PUBLISH_IMAGES_LIST[@]}"
+  do
+    AMEND=""
+    AMEND+=" --amend ${REGISTRY}/${image}:${TAG}-${TRAVIS_TAG}-amd64";
+    AMEND+=" --amend ${REGISTRY}/${image}:${TAG}-${TRAVIS_TAG}-arm64";
+    AMEND+=" --amend ${REGISTRY}/${image}:${TAG}-${TRAVIS_TAG}-ppc64le";
+    AMEND+=" --amend ${REGISTRY}/${image}:${TAG}-${TRAVIS_TAG}-s390x";
+
+    docker manifest create ${REGISTRY}/${image}:"${TAG}-${TRAVIS_TAG}" $AMEND
+    docker manifest push ${REGISTRY}/${image}:"${TAG}-${TRAVIS_TAG}"
+
+    if [[ "${TAG}" != "next" ]]; then
```

Comment:
AFAIK we publish under a given release version.
How will it work when releasing `7.33.0`  for example ?

Reply:
looks like GH action will push `7.33.0` and this will push `latest-travis`... should we also be pushing a `7.33.0-${TRAVIS_TAG}` here too?

Reply:
scripts should not have any TRAVIS related variable at all.

we add 'travis' name in existing variables but that does not require new variables

Reply:
Yes. GH action will push 7.33.0 as well as latest tag, whereas Travis will push 7.33.0-travis and latest-travis tag. 

For releasing 7.33.0, Travis build will be triggered via API call wherein we would pass the version to be released from [release.yml](https://github.com/eclipse-che/che-theia/blob/8c46ce0c20286c26609c3361de2dbf144624f33f/.github/workflows/release.yml#L120).

Reply:
>  whereas Travis will push 7.33.0-travis

I don't see where it will do this ?